mozilla/blurts-server
# Mntor 3319

# References:

Jira: MNTOR-3319

# How to test
- Open the email from the prerequisites;
- Open the “Sign in” CTA;
- Open the data broker link CTA;
- Open the “View dashboard” CTA;
- Observe the URLs of each CTA after arriving on the Monitor dashboard;
- Expected result:
  - For all three cases, after arriving on the dashboard, the URL presents parameters;

Note:  Data broker name link path updated.  Please check [Jira ticket](https://mozilla-hub.atlassian.net/browse/MNTOR-3319)
# Checklist (Definition of Done)
